pkg/users/client.go

```diff
@@ -819,15 +815,15 @@ func (c *Client) CompletePasswordReset(ctx context.Context, opts CompletePasswor
 }
 
 // SendMagicAuthCode creates a one-time Magic Auth code and emails it to the user.
-func (c *Client) SendMagicAuthCode(ctx context.Context, opts SendMagicAuthCodeOpts) (MagicAuthChallengeID, error) {
+func (c *Client) SendMagicAuthCode(ctx context.Context, opts SendMagicAuthCodeOpts) (User, error) {
```

Comment:
I think this is where the downside of calling this "user" exists - I think we should evaluate if this should be "UserID" especially when we call it `ClientID`. 

I probably shouldn't make the final call or anything here, but it just seems confusing that we return a User object but ask people to pass in `id` as `User`

Reply:
Yeah.. I see... but all the other methods have inputs without `Id` prefix. And also, you'll see the difference in the type (`User` vs `string`).

Reply:
This isn't necessarily true in our other packages, so it's something I've been pushing for us to consider further. For example in `MFA` we have the `ID` suffix on all of the passed in values (and the naming convention changes there were the reason we moved from `v1` to `v2` of Go).